canonical-web-and-design/ubuntu.com
# Replace all blog lists with client-side feed

**NB: A review for this is a review for https://github.com/canonical-web-and-design/canonicalwebteam.blog/pull/123 - if you're happy with this please go and approve that**

This replaces all server-side calls to the blog API in the rest of ubuntu.com with client-side calls to `/blog/latest-news`, rounding off the work in https://github.com/canonical-web-and-design/base-squad/issues/714.

QA
--

Go to:

- `/security`
- `/internet-of-things`
- `/telecommunications`
- `/legal/ubuntu-advantage-service-description`
- `/` (homepage)

Check the blog feed lists look the same as they do on live.
